src/styles/base.css

```
@@ -0,0 +1,39 @@
+  body {
+    background-color: var(--color-background);
+    color: var(--color-primary);
+    font-size: var(--text-base);
+    line-height: var(--text-base--line-height);
+  }
+
+  *::selection {
+    background-color: --alpha(var(--color-primary) / 15%);
+  }
+
+  ::view-transition-old(root),
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)